starfish/core/intensity_table/intensity_table_coordinates.py

```diff
@@ -19,15 +19,19 @@ def transfer_physical_coords_from_imagestack_to_intensity_table(
         - Assign those values to the coords arrays for this spot
     """
     # TODO shanaxel42 consider refactoring pixel case where were can just reshape from Imagestack
-    # Add three new coords to xarray (xc, yc, zc)
-    if intensity_table.sizes[Features.AXIS] == 0:
-        return intensity_table
 
     pairs = (
         (Axes.X.value, Coordinates.X.value),
         (Axes.Y.value, Coordinates.Y.value),
         (Axes.ZPLANE.value, Coordinates.Z.value)
     )
+
+    # make sure the intensity table gets empty metadata if there are no intensities
+    if intensity_table.sizes[Features.AXIS] == 0:
+        for axis, coord in pairs:
+            intensity_table[coord] = xr.DataArray(np.zeros((0)), dims='features')
```

Comment:
```suggestion
            intensity_table[coord] = xr.DataArray(np.zeros((0)), dims=Features.AXIS)
```

Reply:
I think this was duplicated; committed 1/2 :-)